helm/goldilocks-app/Chart.yaml

```diff
@@ -1,5 +1,10 @@
 apiVersion: v1
-appVersion: 2.2.0
+appVersion: v3.1.4
+dependencies:
+- condition: vpa.enabled
+  name: vpa
+  repository: https://charts.fairwinds.com/stable
```

Comment:
We have VPA in our catalogue, I think we should reference that
(https://github.com/giantswarm/giantswarm-playground-catalog/blob/master/vertical-pod-autoscaler-app-1.0.3.tgz)